lms/djangoapps/completion/services.py

```diff
@@ -0,0 +1,49 @@
+"""
```

Comment:
Note to self: This is where I am leaving off for now because: 1) I have limited time allocated to this and 2) I may have jumped into this review prematurely.  I'm getting confused around what reviews are needed when, and I should probably wait for the explicit messages from @jcdyer asking for review. :)

Reply:
I'm having trouble juggling all the PRs right now, too.  I'm working toward getting the early ones merged, so there's less to focus on.